pyproject.toml

```diff
@@ -32,18 +32,19 @@ documentation = "https://pyttb.readthedocs.io"
 
 [project.optional-dependencies]
 dev = [
-    "black[jupyter]",
-    "mypy",
-    "nbstripout",
-    "pytest",
-    "pytest-cov",
-    "ruff==0.0.284",
-    "pre-commit",
+    "mypy>=1.10,<1.14.0",
+    # Also in pre-commit
+    "nbstripout>=0.8,<0.9",
+    "pytest>8.0",
```

Comment:
Did something change in a recent version of pytest? In the past running `pytest .` would run all doctest and `tests/test_*.py` tests. Now, I need to run `pytest . tests` to get the `tests/test_*.py` tests to run. It looks like this was updated in the regression GH Action, but not in the docs.

Reply:
I'm not sure. My history on my command line had the tests directory specified. I'll update the doc.

Reply:
According to the pytest docs I'd expect it to still work unless I broke something in our config. I updated the doc but can check more later

Reply:
Looks good.